internal/alert/sync.go

```diff
@@ -280,6 +278,12 @@ func (p *PromMultiClient) GetAlerts(ctx context.Context) ([]promv1.Alert, bool,
 			allAlerts = append(allAlerts, alerts...)
 		}
 	}
+	// do not return errors on partial failures
+	// metrics will surface the issue
+	// and conditions will still reconcile
+	if partial {
```

Comment:
should paritial distinguish between all of the sources failing and at least one source failing?

Reply:
That sounds reasonable, we can do a calculation here to determine if it's truly partial or not.